src/nvim/api/vim.c

```diff
@@ -133,7 +133,7 @@ String vim_command_output(String str, Error *err)
   return cstr_to_string((char *)get_vim_var_str(VV_COMMAND_OUTPUT));
 }
 
-/// Evaluates the expression str using the vim internal expression
+/// Evaluates the expression str using vim's internal expression
```

Comment:
old wording was better. Maybe capitalize `Vim`. Avoid possessive case in general.